stylelint.config.js

```diff
@@ -0,0 +1,21 @@
+// eslint-disable-next-line
```

Comment:
why are we disabling linter here @Goldziher

Reply:
i'll clean this (if its actually, wrong, will check).